api/models/AccessToken.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Access Token to be used for OAuth.
+ */
+
```

Comment:
Added DDL explicitly to all models (and disabled auto migration because Waterline doesn't support composite primary keys and indexes).